LanguageFeatures/Subtyping/dynamic/generated/records_arguments_binding_A01_t01.dart

```diff
@@ -0,0 +1,125 @@
+// Copyright (c) 2018, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+/// @assertion We say that a type T0 is a subtype of a type T1 (written
+/// T0 <: T1) when:
+/// A record type A is a subtype of record type B iff they have same shape and
+/// the types of all fields of A are subtypes of the corresponding field types
+/// of B
+/// 
+/// @description Check that if type T0 is a record with the same shape that T1
+/// and the types of all fields of T0 is subtype of of the corresponding field
+/// types of T1 then T0 is subtype of T1. Check positional fields only
+/// @author sgrekhov22@gmail.com
+///
+/// @description Check that if type T0 is a subtype of a type T1, then instance
+/// of T0 can be used as an argument of type T1
+/// @author sgrekhov@unipro.ru
+///
+/// This test is generated from records_A01.dart and 
+/// arguments_binding_x01.dart.
+/// Don't modify it. If you want to change this test, change one of the files 
+/// above and then run generator.dart to regenerate the tests.
+
```

Comment:
I see some issues in tests generator which produces all of these generated tests. Including extra blank lines. But I'm going to fix it later as a separate change. Ohterwise, any change in test generator affects all 2K generated subtyping tests and this will make review very hard.

Reply:
OK, at that point I hadn't noticed the comment "This file is generated ...".